### ISSUE TITLE
Many .elf won't work on PCSX2

### ISSUE THREAD

**Issue by atomic83GitHub** [PCSX2/pcsx2#2143]
***PCSX2 version:***
Last 1.5 dev

***PCSX2 options:***
All default, speedhacks disabled.

***Plugins used:***
All default

***Description of the issue/Meta:***
Many .elf don't work as expected on pcsx2:

-PS2INFOGB-Crash at the loading (Interpreter + cache don't seems to improve the situation)
![capture](https://user-images.githubusercontent.com/26351275/33138290-674186b8-cfaa-11e7-8327-2479fcf9d98e.PNG)

-Aura for PS2:
Black box in the middle of the screen under GSDX hardware mode:
![capture](https://user-images.githubusercontent.com/26351275/33138766-97cb8788-cfab-11e7-8967-3bf4220bec2e.PNG)

Missing polygons on GSDX hardware mode:
-Hardware:
![capture2](https://user-images.githubusercontent.com/26351275/33138785-a578a7f8-cfab-11e7-91dc-1601d620eb01.PNG)
Software:
![capture3](https://user-images.githubusercontent.com/26351275/33138811-b50bead6-cfab-11e7-8010-f3f805440b61.PNG)

Huge performance impact Under GSDX hardware mode:
Hardware:
![capture4](https://user-images.githubusercontent.com/26351275/33138848-d123ccd4-cfab-11e7-8218-bfa18fd32fa1.PNG)
Software:
![6](https://user-images.githubusercontent.com/26351275/33138867-e1617c9a-cfab-11e7-981b-295079324e69.PNG)

***How to reproduce the issue:***
Load PCSX2 and one of the folowing .elf

***Not a regression***

***PC specifications:***
AMD fx8350 GTX 780Ti Windows 10 64 Bit

***If you want GS/Core dump, say it to me***

***ELF directory***
https://drive.google.com/file/d/1GGj7gbRiq4bf_Bn35vVUrtAGhqxfDStS/view?usp=sharing

**Comment by ramapcsx2** [comment]
We know.
There are a few things that make homebrew elfs harder to emulate.
The biggest problem is their use of an early ps2 sdk, which is very buggy. We found one scene demo, that wrote to a nonexisting root counter register and expected it to work.
By some miracle, real hardware glitched in such a way that it did in fact work.
We implemented a workaround for that demo.

Next problem is, particularly with scene demos. They try to do special things and do not follow Sony recommendations. So what happens is that those features are poorly documented and get rarely seen by us. Hence they stay broken.

Last I want to mention that many homebrews came with specific instructions, many of which are lost today.
Some will expect you to insert a level data DVD or USB stick and you'll never know.

So yeah. We know about it :)
